lib/language_pack/helpers/nodebin.rb

```diff
@@ -0,0 +1,24 @@
+require 'json'
+
+class LanguagePack::Nodebin
+  URL = "https://nodebin.herokai.com/v1/"
+
+  def self.query(q)
+    response = Net::HTTP.get_response(URI("#{URL}/#{q}"))
+    if response.code == '200'
+      JSON.parse(response.body)
+    end
+  end
+
+  def self.node(q)
+    query("node/linux-x64/#{q}")
+  end
+
+  def self.node_lts
+    node("latest?range=6.x")
```

Comment:
I'd be happy to submit a PR too if you're down with it :) I just imagine if say the `Go` buildpack needs to start bundling yarn + node, it would be nice to have a path for it.